api/secvices/user.service.ts

```diff
@@ -0,0 +1,67 @@
+import user from '../modules/user';
+import { Op } from 'sequelize';
+import UserType from '../types/user';
+
+class UserService {
+    public static async getById(id: number): Promise<UserType> {
```

Comment:
Why did you made all methods static? They should be usual methods in service. And service instance should be created in controller.

Reply:
done